experimental/add_to_app/android_using_plugin/app/src/main/java/dev/flutter/example/androidusingplugin/MyApplication.kt

```diff
@@ -5,7 +5,7 @@
 package dev.flutter.example.androidusingplugin
 
 import android.app.Application
-import dev.flutter.plugins.GeneratedPluginRegistrant
+import io.flutter.plugins.GeneratedPluginRegistrant
```

Comment:
The engineering team recently landed a change that allows us to take this reference (and line 33 below) out entirely.

Reply:
*senpai noticed me and he knows that I'm portuguese [shock-face] [screams]*

that's awesome that they landed that change! I updated flutter, removed both lines and could run the project and the plugin worked.

Notes:
- Ran a clean on Android side (the same as `./gradlew clean`)
- Ran the `flutter pub get` in the `flutter_module_using_plugin` folder

Reply:
Also a quick question: isn´t it best to add the 

```
flutter create -t module --org=dev.flutter.example .
```

command to the README in case of people not being able to compile the project?